db/schema/item.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const user = require('./user');
+
+const itemSchema = new mongoose.Schema({
+    id: String,
+    date: String,
+    tax: Number,
+    tip: Number,
+    currency: {
+        type: String,
+        default: 'USD'
+    },
+    total: Number,
+    creator: user,
+    status: String

```

Comment:
Can we add "PENDING" and "PAID" as enums for status, similar to what was defined for the status field in the transaction schema

Reply:
should we go with "PENDING" "PAID" "CANCELLED" or 'COMPLETED','ONGOING','DELETED' ?